google-cloud-pubsub/acceptance/pubsub_helper.rb

```diff
@@ -87,6 +89,19 @@ def clean_up_pubsub_topics
   puts "Error while cleaning up pubsub topics after tests.\n\n#{e}"
 end
 
+def clean_up_pubsub_snapshots
```

Comment:
@quartzmo I was thinking that `clean_up_pubsub_snapshots` should also be added to the `acceptance:cleanup` rake task as well.